Arrangement_on_surface_2/include/CGAL/Arr_counting_traits_2.h

```diff
@@ -966,7 +966,7 @@ class Arr_counting_traits_2 : public Base_traits {
 
   /*! Clean all operation counters */
   void clear_counters()
-  { memset(m_counters, 0, sizeof(m_counters)); }
+  { memset_s(m_counters, sizeof(m_counters), 0, sizeof(m_counters)); }
```

Comment:
@efifogel do you agree that this is not time critical and we can replace with  `std::fill()` ?

Reply:
In addition, you can modernize the code, and replace `m_counters` with a `std::array`.